e2e_test/v2/batch/func.slt

```diff
@@ -69,3 +69,42 @@ drop table t;
 
 statement ok
 drop table b;
+
+query T
+select concat_ws(',', 'a', 'b');
+----
+a,b
+
+query T
+select concat_ws(NULL, 'a', 'b');
+----
+NULL
+
+query T
+select concat_ws(',', NULL, 'b');
+----
+b
+
+query T
+select concat_ws(NULL, NULL, 'b');
+----
+NULL
+
+query T
+select concat_ws(true, 1, 1.01, 'A', NULL);
+----
+1true1.01trueA
+
+statement ok
+create table t (v1 bool, v2 smallint, v3 int, v4 decimal, v5 real, v6 double, v7 varchar, v8 varchar);
+
+statement ok
+insert into t values (true, 1, 2, 3.01, 4, 5.01, 'abc', NULL);
+
+query T
+select concat_ws(v1, v2, v3, v4, v5, v6, v7, v8) from t;
+----
+1true2true3.01true4true5.01trueabc
```

Comment:
These 2 cases are invalid in PostgreSQL. The 0-th arg is required to be string without casting.